README.md

```diff
@@ -0,0 +1,186 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="img/todolist_logo_Readme.JPG" alt="logo" width="50%"  height="auto" />
+  <br/>
+  <h3><b>To Do List - Webpack</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+**To Do List** is a simple HTML list of To Do tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://HTML.org/">HTML</a></li>
+    <li><a href="https://css.org/">CSS</a></li>
+    <li><a href="https://javascript.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Use local Storage**
+- **Save a list of books**
+- **Allow eliminate one selected Book**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+> Comming Soon
+<!-- - [To Do List - Live Demo]() -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+* Only clone the repo and run in your local
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/JuanDiegoGuerra/Awesome-Books-JS.git
+
+### Install
+
+<!-- Install this project with: -->
+
+Clone this repository to your desired folder:
+
+https://github.com/JuanDiegoGuerra/Awesome-Books-JS.git
+
+### Usage
+
+To run the project, execute the following command:
+
+git clone https://github.com/JuanDiegoGuerra/Awesome-Books-JS.git
+
+### Run tests
```

Comment:
- [ OPTIONAL] Kindly update the "Getting started" section with the appropriate information that are related to this project and make sure you use the the link to this repo not the "Awesome-Books" repo. That will make your readme file professional.

Reply:
Thank u mate, I will improve the readme file.